src/adagrams.js

```diff
@@ -1,8 +1,68 @@
 const Adagrams = {
+  startingLetters: ["A", "A", "A", "A", "A", "A", "A", "A", "A", "B","B","C","C","D","D","D","D","E","E","E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "F", "F", "G", "G", "G", "H", "H", "I", "I", "I", "I", "I", "I", "I", "I", "I", "K", "L", "L", "L", "L", "M", "M", "N", "N", "N", "N", "N", "N", "O", "O", "O", "O", "O", "O", "P", "P", "Q", "R", "R","R", "R", "R", "R", "S", "S", "S", "S", "T", "T","T","T","T","T", "U", "U", "U", "U", "V", "V", "W", "W", "X", "Y", "Y", "Z"],
+
   drawLetters() {
-    // Implement this method for wave 1
+    let copyLetters = Adagrams.startingLetters.map(num => num)
+    let playersLetters = [];
+    for (let i = 0; i < 10; i +=1 ) {
+      let index = Math.floor(Math.random() * copyLetters.length)
+      let letter = copyLetters[index]
+      playersLetters.push(letter);
+      copyLetters.splice(index, 1);
+    }
+    return playersLetters;
   },
-};
 
+  usesAvailableLetters(word, lettersInHand) {
+    let handValues = {};
+
+    lettersInHand.forEach(function(letter){
+      if (handValues[letter] == null){
+        handValues[letter] = 1
+      } else {
+        handValues[letter] += 1
+      }
+    });
+
+    let valid = true; 
+
+    word.split('').forEach(function(letter){
```

Comment:
Consider how you could refactor and use a different type of `for` loop to allow you to breakout as soon as you want to return `false`.